app/src/main/res/layout/detail_fragment.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
How would you improve this layout to show the full details of the movie, like the sinopsis (overview) when it doesn't fit the screen?